src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RegisterChecks/RoepaoTests.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Web.Controllers.Roatp.Apply;
+using SFA.DAS.AdminService.Web.Infrastructure;
+using SFA.DAS.AdminService.Web.Services.Gateway;
+using SFA.DAS.AdminService.Web.Validators.Roatp;
+using SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway;
+using SFA.DAS.AssessorService.Api.Types.Models.Validation;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web.Tests.Controllers.Gateway.RegisterChecks
+{
+    [TestFixture]
+    public class RoepaoTests
+    {
+        private RoatpGatewayRegisterChecksController _controller;
+        private Mock<IRoatpApplicationApiClient> _applyApiClient;
+        private Mock<IHttpContextAccessor> _contextAccessor;
+        private Mock<IRoatpGatewayPageViewModelValidator> _gatewayValidator;
+        private Mock<IGatewayRegisterChecksOrchestrator> _orchestrator;
+        private Mock<ILogger<RoatpGatewayRegisterChecksController>> _logger;
+
+        private string username = "john smith";
+        private string givenName = "john";
+        private string surname = "smith";
+
+        [SetUp]
+        public void Setup()
+        {
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _contextAccessor = new Mock<IHttpContextAccessor>();
+            _gatewayValidator = new Mock<IRoatpGatewayPageViewModelValidator>();
+            _logger = new Mock<ILogger<RoatpGatewayRegisterChecksController>>();
+            _orchestrator = new Mock<IGatewayRegisterChecksOrchestrator>();
+
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+             {
+                new Claim(ClaimTypes.NameIdentifier, "1"),
+                new Claim("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn", username),
+                new Claim(ClaimTypes.GivenName, givenName),
+                new Claim(ClaimTypes.Surname, surname)
+             }));
+
+            _contextAccessor.Setup(_ => _.HttpContext).Returns(new DefaultHttpContext { User = user });
+
+            _gatewayValidator.Setup(v => v.Validate(It.IsAny<RoepaoPageViewModel>()))
+                .ReturnsAsync(new ValidationResponse
+                {
+                    Errors = new List<ValidationErrorDetail>()
+                });
+
+            _controller = new RoatpGatewayRegisterChecksController(_applyApiClient.Object, _contextAccessor.Object, _gatewayValidator.Object, _orchestrator.Object, _logger.Object);
+        }
+
+        [Test]
+        public async Task check_roepao_request_is_sent()
```

Comment:
I don't think this test is needed as it'd be impossible for the subsequent test to pass if the orchestrator wasn't called.

Reply:
True, however the other tests follow the same pattern 🤷‍♂